### PR TITLE
LPS-121997 Update service.xml and add Upgrade processes for Country

### PR DESCRIPTION
**Pull request by alee8888**
@drewbrokke @pei-jung

https://issues.liferay.com/browse/LPS-121997

### CODE REVIEW

**Comment by alee8888** [comment]
ci:test:sf

**Comment by drewbrokke** [comment]
ci:test:relevant

File: portal-impl/src/com/liferay/portal/service.xml [hunk 1]
```diff
@@ -289,21 +289,41 @@
...
+		<column name="modifiedDate" type="Date" />
+
 		<!-- Other fields -->
 
-		<column name="name" type="String" />
+		<column name="active" type="boolean" />
 		<column name="a2" type="String" />
 		<column name="a3" type="String" />
-		<column name="number" type="String" />
+		<column name="billingAllowed" type="boolean" />
```
Comment (drewbrokke) [inline portal-impl/src/com/liferay/portal/service.xml:311]: I think that `a2` and `a3` should be sorted before `active`. I'll double-check with Brian.

**Comment by drewbrokke** [comment]
See https://github.com/drewbrokke/liferay-portal/pull/1228

**Pull request closed without merging**